vector-op-types.csv

```diff
@@ -1,50 +1,50 @@
+"Base + [F,D]",VFEQ.[H.S.D],compare,2,0,"1 if ==, else 0",F,I,I,ILLEGAL,First Element,Splat
 Vtypes,VSGE,compare,2,0,"1 if >=, else 0","S/U,F",I,I,ILLEGAL,First Element,Splat
 Base,VSGE.X,compare,2,0,"1 if >=, else 0",S/U,I,I,ILLEGAL,First Element,Splat
-Base,"VFLE.[H,S,D]",compare,2,0,"1 if <=, else 0",F,I,I,ILLEGAL,First Element,Splat
+"Base + [F,D]","VFLE.[H,S,D]",compare,2,0,"1 if <=, else 0",F,I,I,ILLEGAL,First Element,Splat
 Vtypes,VSLT,compare,2,0,"1 if <, else 0","S/U,F",ן,I,ILLEGAL,First Element,Splat
 Base,VSLT.X,compare,2,0,"1 if <, else 0",S/U,ן,I,ILLEGAL,First Element,Splat
-Base,"VFLT.[H,S,D]",compare,2,0,"1 if <, else 0",F,ן,I,ILLEGAL,First Element,Splat
+"Base + [F,D]","VFLT.[H,S,D]",compare,2,0,"1 if <, else 0",F,ן,I,ILLEGAL,First Element,Splat
 Vtypes,VSNE,compare,2,0,"1 if <>, else 0","S/U,F",ן,I,ILLEGAL,First Element,Splat
 Base,VSNE.X,compare,2,0,"1 if <>, else 0",S/U,ן,I,ILLEGAL,First Element,Splat
-Base,"VFNE.[H,S,D]",compare,2,0,"1 if <>, else 0",F,ן,I,ILLEGAL,First Element,Splat
+"Base + [F,D]","VFNE.[H,S,D]",compare,2,0,"1 if <>, else 0",F,ן,I,ILLEGAL,First Element,Splat
 Vtypes,VCVT,convert,1,1,convert type,"S/U,F","F,S/U",ILLEGAL,ILLEGAL,First Element,Splat
-Base,"VFCVT.[X,H,S,D].[X,H,S,D]",convert,1,1,convert type,"S/U,F","F,S/U",ILLEGAL,ILLEGAL,First Element,Splat
+"Base + [F,D]","VFCVT.[X,H,S,D].[X,H,S,D]",convert,1,1,convert type,"S/U,F","F,S/U",ILLEGAL,ILLEGAL,First Element,Splat
 Vtypes,VSGNJ,copy/sign,2,0,FP sign injection,F,F,ILLEGAL,ILLEGAL,First Element,Splat
-Base,"VFSGNJ.[H,S,D]",copy/sign,2,0,FP sign injection,F,F,ILLEGAL,ILLEGAL,First Element,Splat
+"Base + [F,D]","VFSGNJ.[H,S,D]",copy/sign,2,0,FP sign injection,F,F,ILLEGAL,ILLEGAL,First Element,Splat
 Vtypes,VSGNJN,copy/sign,2,0,FP inverted sign injection,F,F,ILLEGAL,ILLEGAL,First Element,Splat
-Base,"VFSGNJN.[H,S,D]",copy/sign,2,0,FP inverted sign injection,F,F,ILLEGAL,ILLEGAL,First Element,Splat
+"Base + [F,D]","VFSGNJN.[H,S,D]",copy/sign,2,0,FP inverted sign injection,F,F,ILLEGAL,ILLEGAL,First Element,Splat
 Vtypes,VSGNJX,copy/sign,2,0,FP xor sign,F,F,ILLEGAL,ILLEGAL,First Element,Splat
-Base,"VFSGNJX.[H,S,D]",copy/sign,2,0,FP xor sign,F,F,ILLEGAL,ILLEGAL,First Element,Splat
-Vtypes,VDIV,divsqrt,2,0,divide,F,F,ILLEGAL,ILLEGAL,First Element,Splat
-Base,"VFDIV[H,S,D]",divsqrt,2,0,divide,F,F,ILLEGAL,ILLEGAL,First Element,Splat
-Vtypes,VREM,divsqrt,2,0,partial remainder,F,F,ILLEGAL,ILLEGAL,First Element,Splat
-Base,"VFREM.[H,S,D]",divsqrt,2,0,partial remainder,F,F,ILLEGAL,ILLEGAL,First Element,Splat
-Vtypes,VSQRT,divsqrt,1,0,square root,F,F,ILLEGAL,ILLEGAL,First Element,Splat
-Base,"VFSQRT.[H,S,D]",divsqrt,1,0,square root,F,F,ILLEGAL,ILLEGAL,First Element,Splat
+"Base + [F,D]","VFSGNJX.[H,S,D]",copy/sign,2,0,FP xor sign,F,F,ILLEGAL,ILLEGAL,First Element,Splat
+Vtypes + M,VDIV,divsqrt,2,0,divide,F,F,ILLEGAL,ILLEGAL,First Element,Splat
+"Base + M + [F,D]","VFDIV[H,S,D]",divsqrt,2,0,divide,F,F,ILLEGAL,ILLEGAL,First Element,Splat

```

Comment:
This doesn't match the scalar spec where FDIV is always present in F, ditto for `VFREM` and `VFSQRT` below.